pkg/payerreport/store.go

```diff
@@ -466,8 +478,12 @@ func prepareStoreReportParams(
 		return nil, ErrEndMinuteSinceEpochTooLarge
 	}
 
+	if len(activeNodeIDs) == 0 {
+		return nil, ErrNoActiveNodeIDs
+	}
```

Comment:
I understand the decision of including this check, but can this happen at all? If there are no active nodes it shouldn't be possible to publish messages, and there won't even be nodes attesting to anything.

Reply:
It’s possible but not likely. There could have been some active nodes, we publish messages, then we kick them all out and try to publish a payer report.